paasta_tools/cli/cmds/spark_run.py

```diff
@@ -587,6 +636,8 @@ def configure_and_run_docker_container(
 
     volumes.append("%s:rw" % args.work_dir)
     volumes.append("/nail/home:/nail/home:rw")
+    if args.enable_k8s_autogen:
+        volumes.append(f"{POD_TEMPLATE_PATH}:{POD_TEMPLATE_PATH}:rw")
```

Comment:
Should this be `:ro` instead? Not that it makes a big difference, feel free to drop.

Reply:
yeah, this one i'll just leave it :P